.cirrus/publish-macos-silicon.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -ex
+
+brew install sbt
+
+git clone --branch try-cirrus-ci https://github.com/scalacenter/sbt.git
```

Comment:
After we merge sbt/sbt#7149, we will replace this line with:

```suggestion
git clone --branch 1.8.x https://github.com/sbt/sbt.git
```

Reply:
Done